### DISCUSSION TITLE
has something changed to supabase auth?

### DISCUSSION THREAD

**Discussion by EdgarsLv** [supabase/supabase#6576] [Questions]
Token valid time 3600.

If pc goes to sleep and in this time token expires it doesn't auto refreshes and even doesn't refreshes when i refresh page but instead i get this error that never happened before.
![image](https://user-images.githubusercontent.com/56837461/165372702-b92a8b39-ce5f-454d-a1ad-6087c1044ab5.png)

Before this happened only when pc was in sleep for many hours but still after page refresh token refreshed.

**Comment by GaryAustin1** [comment]
Possibly two related issues.  
https://github.com/supabase/supabase/issues/6464
https://github.com/supabase/supabase/issues/6452
Some work is going on to address these.
I don't know that either though are related to something recently changing...

**Reply by EdgarsLv** [reply]
Was keeping tab open.
When token had to be refreshed got POST error.
Seconds latter received REFRESH and  SIGNED_IN event.
![image](https://user-images.githubusercontent.com/56837461/165392149-37f7085f-b4ae-435f-b4ce-e0bd7e3b489c.png)

![image](https://user-images.githubusercontent.com/56837461/165392271-858b8a45-9512-43f7-a11f-0387d471add2.png)

And this is only happening in new project i created today. Old project still acts like before, fails to refresh after long sleep period.

Ps. Why is everyone trying to hide supabase project url? Anyway everyone can see it in network tab.

**Reply by GaryAustin1** [reply]
On your PS...  If I'm the one you noticed that blocked it, I don't like posting my Supabase URL in public because then the whole world of lurkers knows it without even trying and can poke and prod if desired.  Also most of the time here it is a test site that may or may not have RLS set.  
It is one thing that a determined bad actor finds my site and the url from using it, versus me advertising it for all to see.

**Comment by EdgarsLv** [comment]
I was building my project with supabase for about 2 months and auth already was not perfect. But recently there is something really fcked up for auth.  Just roll it back...

**Reply by GaryAustin1** [reply]
If you think something is broken recently, then you probably should post an issue with as much info as possible including version that works and version that does not in the issues area.  Not sure a dev from Supabase will necessarily find your discussion.

It is not clear on your post here what your environment is, if you are using auth-helpers https://github.com/supabase-community/supabase-auth-helpers or just the basic client code.
